app/components/conceptsService/concepts.service.spec.js

```diff
@@ -145,7 +155,48 @@ describe('Concept dictionary controllers', function() {
         }));
         it('parseNames should return table of names', inject(function(conceptsService){
             var names = conceptsService.parseNames(validLocalizedConcepts);
-            expect(names.length).toEqual(12);
+            expect(names.length).toEqual(10);
+            
+            var synonym2 = findNameObjectByName("synonym22", names);
+            expect(synonym2.localePreferred).toEqual(true);
+            expect(synonym2.locale).toEqual("es");
+            
+            var fullname2 = findNameObjectByName("fullname2", names);
+            expect(fullname2.localePreferred).toEqual(false);
+            expect(fullname2.conceptNameType).toEqual("FULLY_SPECIFIED");
+        }));
+        
+        it('parseNames should throw error', inject(function(conceptsService){
+           expect(function(){conceptsService.parseNames(invalidLocalizedConcepts)}).toThrow("No fully specified name is present!");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/openmrs/openmrs-owa-conceptdictionary/file/2339254613/issues/source?bid=3196969&fileBranchId=3209439#l170)